### PR TITLE
Release notes doc-12-13-2024

### PR DESCRIPTION
**Pull request by adutta-newrelic**
- Updated the release notes for December 13, 2024.
- Merged the changes from the PR (https://github.com/newrelic/docs-website/pull/19462)
- https://new-relic.atlassian.net/browse/NR-350344

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+subject: "Docs"
+releaseDate: '2024-12-13'
+version: 'version: November 29-December 13, 2024'
+---
+
+### New docs
+
+* [Logging with the iOS agent](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-remote-logging/) introduces enhanced logging capabilities in the iOS agent, including log level configuration and automatic log collection features.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:9]: introduces enhanced logging APIs...

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+* The [Kubernetes APM auto-attach](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator), formerly known as the Kubernetes agent operator, now streamlines full-stack observability for Kubernetes environments by automating APM instrumentation alongside Kubernetes agent deployment.
+* [Forward AWS services logs to New Relic](https://docs.newrelic.com/install/aws-logs/?service=aws_services) now include a new log integration method using a CloudFormation template to forward logs, customize triggers, and configure endpoints from New Relic One, in addition to existing methods.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:11]: * [Forward AWS services logs to New Relic](https://docs.newrelic.com/install/aws-logs/?service=aws_services)  includes a new log integration method using a CloudFormation template to forward logs.

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+### Major changes
+
+* Updated the version support information (under prerequisites section) about the [service broker for TAS](https://docs.newrelic.com/docs/infrastructure/other-infrastructure-integrations/cloudfoundry-integrations/vmware-tanzu-service-broker-integration/).
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:15]: What is TAS? Didn't find any reference

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+* The [synthetic job manager](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager/) now supports Podman container environment.
+* There is an update about [Node.js agent compatibility](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/).   
+* We've deprecated the Snowflake integration with New Relic. Refer the [topic](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration/) to understand on usage of Snowflake.
+* Updated [JMX monitoring integration](https://docs.newrelic.com/install/jmx/?dropdown1=linux&deployment=apt) to include instructions for setting `JAVA_HOME` when Java is not found on the PATH.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:19]: use contraction: ...when Java isn't...

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+* The [Kubernetes APM auto-attach](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator), formerly known as the Kubernetes agent operator, now streamlines full-stack observability for Kubernetes environments by automating APM instrumentation alongside Kubernetes agent deployment.
+* [Forward AWS services logs to New Relic](https://docs.newrelic.com/install/aws-logs/?service=aws_services) now include a new log integration method using a CloudFormation template to forward logs, customize triggers, and configure endpoints from New Relic One, in addition to existing methods.
+
+### Major changes
+
+* Updated the version support information (under prerequisites section) about the [service broker for TAS](https://docs.newrelic.com/docs/infrastructure/other-infrastructure-integrations/cloudfoundry-integrations/vmware-tanzu-service-broker-integration/).
+* The [synthetic job manager](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager/) now supports Podman container environment.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:16]: Make sentences parallel.
Alternate: Introduced Podman  environment support in the [synthetic job manager](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager/)

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+* There is an update about [Node.js agent compatibility](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/).
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:17]: Alternate: Updated [Node.js agent compatibility](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/).

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+* [Forward AWS services logs to New Relic](https://docs.newrelic.com/install/aws-logs/?service=aws_services) now include a new log integration method using a CloudFormation template to forward logs, customize triggers, and configure endpoints from New Relic One, in addition to existing methods.
+
+### Major changes
+
+* Updated the version support information (under prerequisites section) about the [service broker for TAS](https://docs.newrelic.com/docs/infrastructure/other-infrastructure-integrations/cloudfoundry-integrations/vmware-tanzu-service-broker-integration/).
+* The [synthetic job manager](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager/) now supports Podman container environment.
+* There is an update about [Node.js agent compatibility](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/).   
+* We've deprecated the Snowflake integration with New Relic. Refer the [topic](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration/) to understand on usage of Snowflake.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:18]: Remove we've

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx (hunk 3 above)
Comment (adutta-newrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:15]: Need to check this with Chet

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+* Introduced [Automatically observe page resource assets](https://docs.newrelic.com/docs/browser/new-relic-browser/browser-pro-features/page-resources/) and updated [Experimental features in browser monitoring](https://docs.newrelic.com/docs/browser/new-relic-browser/configuration/experimental-features/) pages to detail new experimental features for tracking browser performance events with New Relic Browser.
+
+### Minor changes
+
+* New Relic extended end of life support for [Azure SQL database integration polling API](https://docs.newrelic.com/eol/2024/04/eol-04-24-24-polling-api/) to July 31, 2025.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:26]: You can remove New Relic

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+* Updated [Ruby agent requirements and supported frameworks](http://docs.newrelic.com/docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks/#http_clients) to include support for additional message broker gem.
+* In [EntitiesByDomainTypeQuery]((https://docs.newrelic.com/docs/new-relic-solutions/build-nr-ui/sdk-component/query-and-storage/EntitiesByDomainTypeQuery/)), `sortBy` attribute is replaced by `sortType` for fetching sorting criteria.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:28]: Make the sentence parallel.

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx [hunk 10]
```diff
@@ -0,0 +1,127 @@
+---
+subject: "Docs"
+releaseDate: '2024-12-13'
+version: 'version: November 29-December 13, 2024'
```
[the remark this answers is not in the document]
Comment (adutta-newrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:4]: Updated

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx (hunk 1 above)
Comment (adutta-newrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:9]: Updated

Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:9]: Change the link to` /docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-remote-logging/` from `https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-remote-logging/`

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx (hunk 2 above)
Comment (adutta-newrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:11]: Updated

**Review by cbehera-newrelic — COMMENTED** [review]
Minor enhancements are suggested. Also, ensure that you have maintained parallelism throughout .....

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx (hunk 5 above)
Comment (adutta-newrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:16]: Updated

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx (hunk 7 above)
Comment (adutta-newrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:18]: Updated

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx (hunk 4 above)
Comment (adutta-newrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:19]: Updated

File: src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx (hunk 1 above)
Comment (adutta-newrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx:9]: Updated

**Pull request merged**